src/resources/troubleshooting.md

```diff
@@ -0,0 +1,720 @@
+This can be mainly due to the fact that the leaderboard has a large database (i.e. the total number of users with a modified variable value other than the default).
+
+---
+## Bot Issues
+### The Bot Is Offline
+**The 1st Reason**\
+The node is restarting. While the node is restarting, logically, the bot can't work.\
+Usually restarting doesn't take more than 5-10 minutes.\
+So please wait a while and have a cup of tea or coffee while the node restarts.
+
+**The 2nd Reason**\
+For some reason, bot's token is no longer valid or the BDFD app thinks so.\
+You can solve this problem by regenerating bot's token on the developer portal and then replacing the old token in the BDFD app with the new one in the bot's Settings tab.
+
+**The 3rd Reason**\
+Not a common problem, but possible. The node your bot is running on is experiencing problems.\
+In this case, join the [support server](https://discord.gg/botdesigner), create a ticket using the `!new` command and tell the staff your bot's ID and node number, if you know it (node number can be found out using the [`$botNode`](../bdscript/botNode.md) function when your bot is online).\
+The staff will inform the developers of the current problems, providing the scale of the problem (affected bots and/or nodes).\
+**Please don't regenerate bot token in this case**, as it leads to changing the node, and if everyone starts changing their node from problem node to healthy node, healthy node can also be affected by the problem.
+
+
+### The Bot Doesn't Respond
+**The 1st Reason**\
+If your bot is based on text commands, and if you have Message Content Intent disabled, you must enable it in your bot's settings.\
+Read the [Gateway Intents Guide](../guides/gatewayIntents.md) for more details.
+
+**The 2nd Reason**\
+Your bot doesn't have the necessary permissions.\
+In order for your bot to respond correctly to a command, it must have permissions for `Send Messages`, `Embed Links` (if your code has embed functions), and `Read Messages` in order for the bot to have access to the channel.
+
+### The Bot Goes Offline From Time To Time
+This is due to the fact that nodes are restarted from time to time to maintain stable operation of all the bots, which also work on this node.
+
+### Desynchronization of Commands
+Desynchronization of commands means, for example, that you have deleted a command but the bot still responds to it (aka. Ghost Command), or you see different code of your command in the web app and another in the mobile app.
+#### Ghost Command
+Not many people encounter this problem, but it's still worth mentioning.\
+This problem is related to database synchronization (between your application data and your bot's data in the database).\
+**Solution Options**\
+*The 1st Solution*\
+Restart the app.\
+Close the app from Recent apps and reopen it. This may result in a deleted command that the deletion didn't sync with the database, and then you can delete that command again.\
```

Comment:
```suggestion
Close the app from Recent apps and reopen it. This may result in a deleted command that the deletion didn't sync with the database, and then you can delete that command again.

```